src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Serialization/MessagePack/Formatters/RazorConfigurationFormatter.cs

```diff
@@ -22,8 +22,9 @@ public override RazorConfiguration Deserialize(ref MessagePackReader reader, Ser
 
         var configurationName = CachedStringFormatter.Instance.Deserialize(ref reader, options) ?? string.Empty;
         var languageVersionText = CachedStringFormatter.Instance.Deserialize(ref reader, options) ?? string.Empty;
+        var suppressAddComponentParameter = reader.ReadBoolean();
 
-        count -= 2;
+        count -= 3;
 
         if (reader.NextMessagePackType is MessagePackType.Boolean)
```

Comment:
Can this check for backward-compatibility be removed now that you're bumping the serialization format version? Since `forceRuntimeCodeGeneration` isn't written by `Serailize(...)` below, I don't think there's a way to get into a situation where there's a MessagePack-formatter `RazorConfiguration` that has both `SuppressAddComponentParameter` and `ForceRuntimeCodeGeneration`.